content/src/service/garbage-collection/GarbageCollectionManager.ts

```diff
@@ -1,12 +1,12 @@
 import { ILoggerComponent } from '@well-known-components/interfaces'
-import { ContentFileHash, delay } from 'dcl-catalyst-commons'
+import { delay } from 'dcl-catalyst-commons'
```

Comment:
Don't we have any better place where to get a `delay` function from? In my next PR I'm removing the dependency of `dcl-catalyst-commons`...